### PR TITLE
Add destroy, disable, enable ops for azure server groups

### PR DESCRIPTION
**Pull request by milismsft**

### CODE REVIEW

**Comment by milismsft** [comment]
@spinnaker/netflix-reviewers PTAL

**Comment by milismsft** [comment]
@anotherchrisberry any feedback? Would you be able to merge it?

File: app/scripts/modules/azure/serverGroup/serverGroup.write.service.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        description: 'Enable Server Group: ' + serverGroup.name
+      });
+    }
+
+    return {
+      destroyServerGroup: destroyServerGroup,
+      disableServerGroup: disableServerGroup,
+      enableServerGroup: enableServerGroup
+    };
+  });
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/serverGroup/serverGroup.write.service.js]: This looks almost identical to https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/serverGroup/serverGroup.write.service.js - the only difference I see is that `destroyServerGroup` doesn't include the `region` param.

Can you not use the existing service for some reason?

Comment (milismsft) [inline app/scripts/modules/azure/serverGroup/serverGroup.write.service.js]: There're also settings in the core file for zone param that I was not sure what downfall impact might have.

**Comment by anotherchrisberry** [comment]
LGTM

**Pull request merged**